content/post/2017-04-28-will-landau-s-portfolio-for-minimalists.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Will Landau's portfolio: a website for minimalists"
+author: Will Landau
+date: 2017-04-28
+slug: landau-website
+tags: [slim, syaning, jekyll]
```

Comment:
I forgot to mention that it would great if you could add a `description` here in YAML.